ott/convert/csv2json/gtfs.py

```diff
@@ -122,16 +123,34 @@ def gtfs_feed_parser():
             parser.print_help()
 
 
+def cat_agency_data():
```

Comment:
this is a routine that digs into a folder of GTFS.zip files, and reads each agency.txt file, and outputs the agency(s) therein.

can be used for the config for OTP-RR route listings

Reply:
so many different config files...